### PR TITLE
Update: Add dts in confirmation modal and list picker components

### PR DESCRIPTION
**Pull request by NamrataTrivedi**
Adding dts in in confirmation modal and list picker componenets, so it can be used in e2e tests.

### CODE REVIEW

File: src/components/adslotUi/ConfirmModalComponent.js [hunk 1]
```diff
@@ -25,8 +25,9 @@ const ConfirmModalComponent = ({
         <p>{modalDescription}</p>
       </Modal.Body>
       <Modal.Footer>
-        {modalClose ? <Button className="btn-inverse" onClick={cancelAction}>{buttonCancelLabel}</Button> : null}
-        <Button bsStyle="primary" onClick={applyAction}>{buttonConfirmLabel}</Button>
+        {modalClose ? <Button className="btn-inverse" onClick={cancelAction} data-test-selector="modal-cancel">
```
Comment (OndrejCholeva) [inline src/components/adslotUi/ConfirmModalComponent.js]: `confirmation-modal...`

File: package.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
   ],
   "author": "Adslot",
   "devDependencies": {
-    "alexandria-adslot": "^2.5.0",
+    "alexandria-adslot": "https://github.com/Adslot/alexandria.git#5822f6563cab6eb3b64eae7f8fba41e96795e411",
```
Comment (omgaz) [inline package.json]: Released: https://github.com/Adslot/alexandria/releases/tag/v2.5.1

**Comment by omgaz** [comment]
LGTM :)

**Comment by davepoon** [comment]
LGTM :thumbsup:

**Comment by NamrataTrivedi** [comment]
@omgaz @davepoon merge?

**Pull request merged**